source/getting-started/first-app.md

```diff
@@ -7,28 +6,29 @@ ember new my-new-app
 
 This will create a new `my-new-app` directory with your new Ember app inside.
 
-Once the generation process finishes, launch the Ember CLI development server:
+Once the process finishes, launch the Ember development server:
```

Comment:
I think we need some sort of consistency around when we say "Ember" and when we say "Ember CLI". Thoughts? /cc @trek @rwjblue @locks @tomdale